blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/server/model/BbServerUser.java

```diff
@@ -14,16 +14,14 @@
  * @author Vivek Pandey
  */
 public class BbServerUser extends BbUser {
-    private final String name;
     private final String displayName;
     private final String slug;
     private final String emailAddress;
     private final String avatar;
 
     @JsonCreator
-    public BbServerUser(@Nonnull @JsonProperty("name") String name, @Nonnull @JsonProperty("displayName") String displayName,
+    public BbServerUser(@Nonnull @JsonProperty("displayName") String displayName,
```

Comment:
No changes in server APIs, we should not change this.

Reply:
You suggested we remove getName() (as its not used anywhere). The constructor isn't data bound, so I figured it was safe to remove the field.